gdx/src/com/badlogic/gdx/graphics/glutils/GLFrameBuffer.java

```diff
@@ -174,14 +182,33 @@ private void build () {
 
 		int result = gl.glCheckFramebufferStatus(GL20.GL_FRAMEBUFFER);
 
+		if (result == GL20.GL_FRAMEBUFFER_UNSUPPORTED && hasDepth && hasStencil
+			&& gl.glGetString(GL20.GL_EXTENSIONS).contains("GL_OES_packed_depth_stencil")) {
```

Comment:
Can you change this to [`Gdx.graphics.supportsExtension`](https://libgdx.badlogicgames.com/nightlies/docs/api/com/badlogic/gdx/Graphics.html#supportsExtension-java.lang.String-)?